### ISSUE TITLE
7.2.0 / typist does not respect pagination

### ISSUE THREAD

**Issue by scalliondelight** [JujuAdams/Scribble#219]
I noticed in the documentation that you state typist will be reset when changing pages, but is it supposed to return to pos 0 for the entire element when switching pages? this causes typist to repeat ticks for glyphs that are no longer visible (as they are on a previous page which is not being displayed). forgive me if I'm doing something wrong or misunderstanding, but this seems like weird behavior. I've tried a few different phrasings, but no dice. 

```
if obj_Player.confirmed {	
        if  typist.get_state() >= 1 {
		var thisPage = thisText.get_page();
		debug_log("advancing page", thisPage+1);
		thisText.page(thisPage+1);
		typist.in(.25, 0);
	}
	else typist.in(10, 0);
}
debug_log(typist.get_position(), "typist position");
```

in the code above, I'm allowing the player to advance typist more quickly if they register input. If typist has completed the current page, then input will instead advance the page. When this occurs, typist begins to print again but it will print blank characters for `n` characters (where `n` characters == the number of characters on page 0) before printing the characters on page 1. it doesn't actually physically advance the typist on the screen, but it executes sounds and that debug line shows typist advancing. am I holding it wrong?

**Comment by JujuAdams** [comment]
Known issue! Will hopefully work on it this week

**Comment by JujuAdams** [comment]
_One month later_

Ok actually looking at this now

**Comment by JujuAdams** [comment]
k, think this is fixed for v7.2.0pre002
